src/relayFeeCalculator/chain-queries/alephZero.ts

```diff
@@ -1,36 +1,10 @@
-import assert from "assert";
-import { getDeployedAddress } from "../../utils/DeploymentUtils";
-import { DEFAULT_LOGGER, Logger } from "../relayFeeCalculator";
-import { providers } from "ethers";
-import { CHAIN_IDs, DEFAULT_SIMULATED_RELAYER_ADDRESS, TOKEN_SYMBOLS_MAP } from "../../constants";
+import { CHAIN_IDs } from "../../constants";
 import { Coingecko } from "../../coingecko/Coingecko";
-import { isDefined } from "../../utils";
 import { QueryBase } from "./baseQuery";
 
 // @dev This class only exists because querying the native token price for this network is not the standard
 // CoinGecko query because the native token symbol is not ETH.
 export class AlephZeroQueries extends QueryBase {
-  constructor(
-    provider: providers.Provider,
-    symbolMapping = TOKEN_SYMBOLS_MAP,
-    spokePoolAddress = getDeployedAddress("SpokePool", CHAIN_IDs.ALEPH_ZERO),
-    simulatedRelayerAddress = DEFAULT_SIMULATED_RELAYER_ADDRESS,
-    coingeckoProApiKey?: string,
-    logger: Logger = DEFAULT_LOGGER
-  ) {
-    assert(isDefined(spokePoolAddress));
-    super(
-      provider,
-      symbolMapping,
-      spokePoolAddress,
-      simulatedRelayerAddress,
-      logger,
-      coingeckoProApiKey,
-      undefined,
-      "usd"
```

Comment:
This `coingeckoBaseCurrency` class variable is only used in the `QueryBase`'s `getTokenPrice()` function which gets overridden by this class anyways, so its not useful to pass through. The only argument I see to keeping this code here is to prevent a future developer from accidentally using `this.coingeckoBaseCurrency` within this class and using an unexpected currency